src/qml/qmltablefromlistmodelcolumn.h

```diff
@@ -0,0 +1,164 @@
+    void setSetStatusTip(const QJSValue& function);
+    QJSValue whatsThis() const;
+    void setWhatsThis(const QJSValue& stringOrFunction);
+    QJSValue getSetWhatsThis() const;
+    void setSetWhatsThis(const QJSValue& function);
+    QJSValue font() const;
+    void setFont(const QJSValue& stringOrFunction);
+    QJSValue getSetFont() const;
+    void setSetFont(const QJSValue& function);
+    QJSValue textAlignment() const;
+    void setTextAlignment(const QJSValue& stringOrFunction);
+    QJSValue getSetTextAlignment() const;
+    void setSetTextAlignment(const QJSValue& function);
+    QJSValue background() const;
+    void setBackground(const QJSValue& stringOrFunction);
+    QJSValue getSetBackground() const;
+    void setSetBackground(const QJSValue& function);
+    QJSValue foreground() const;
+    void setForeground(const QJSValue& stringOrFunction);
+    QJSValue getSetForeground() const;
+    void setSetForeground(const QJSValue& function);
+    QJSValue checkState() const;
+    void setCheckState(const QJSValue& stringOrFunction);
+    QJSValue getSetCheckState() const;
+    void setSetCheckState(const QJSValue& function);
+    QJSValue accessibleText() const;
+    void setAccessibleText(const QJSValue& stringOrFunction);
+    QJSValue getSetAccessibleText() const;
+    void setSetAccessibleText(const QJSValue& function);
+    QJSValue accessibleDescription() const;
+    void setAccessibleDescription(const QJSValue& stringOrFunction);
+    QJSValue getSetAccessibleDescription() const;
+    void setSetAccessibleDescription(const QJSValue& function);
+    QJSValue sizeHint() const;
+    void setSizeHint(const QJSValue& stringOrFunction);
+    QJSValue getSetSizeHint() const;
+    void setSetSizeHint(const QJSValue& function);
+    QJSValue getterAtRole(const QString& roleName);
+    QJSValue setterAtRole(const QString& roleName);
+    const QHash<QString, QJSValue> getters() const;
+    static const QHash<int, QString> supportedRoleNames();
+  Q_SIGNALS:
+    void indexChanged();
+    void displayChanged();
+    void setDisplayChanged();
+    void decorationChanged();
+    void setDecorationChanged();
+    void editChanged();
+    void setEditChanged();
+    void toolTipChanged();
+    void setToolTipChanged();
+    void statusTipChanged();
+    void setStatusTipChanged();
+    void whatsThisChanged();
+    void setWhatsThisChanged();
+    void fontChanged();
+    void setFontChanged();
+    void textAlignmentChanged();
+    void setTextAlignmentChanged();
+    void backgroundChanged();
+    void setBackgroundChanged();
+    void foregroundChanged();
+    void setForegroundChanged();
+    void checkStateChanged();
+    void setCheckStateChanged();
+    void accessibleTextChanged();
+    void setAccessibleTextChanged();
+    void accessibleDescriptionChanged();
+    void setAccessibleDescriptionChanged();
+    void sizeHintChanged();
+    void setSizeHintChanged();
+
+  private:
+    int mIndex = -1;
```

Comment:
```suggestion
    int m_index = -1;
```
The same applies to all members.